src/foundation.cs

```diff
@@ -11643,11 +11643,13 @@ interface NSPort : NSCoding, NSCopying {
 	[Model, BaseType (typeof (NSObject))]
 	[Protocol]
 	interface NSPortDelegate {
```

Comment:
Any reason why `[NoMacCatalyst]` is instead on the parent delegate?

Reply:
it was noted in the `.ignore` file - that I forgot to commit :|